src/session/isAppSessionQuery/isAppSessionQuery.ts

```diff
@@ -1,19 +1,12 @@
 import { AppSessionQuery } from '../types'
+import { hasKey } from '../../utils/hasKey/hasKey'
```

Comment:
didn't you add `utils/index.ts` to clean things like this?

Reply:
Good point! will do